### PR TITLE
skipnav: a tragedy in 3 acts

### PR DESCRIPTION
**Pull request by braxex**
## Description

skipnav was bein weird. not anymore tho.

### How to test

go check out the skipnav on report page and non-report page. it's nice, yes?

### Changed Dependencies

nope

## Code author checklist
- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://bit.ly/3zPrxuZ) tests
- [x] ~~I have added analytics, if necessary~~
- [x] ~~I have updated the documentation, if necessary~~

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

File: services/ui-src/src/components/app/SkipNav.test.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { render, screen } from "@testing-library/react";
-import userEvent from "@testing-library/user-event";
+// import userEvent from "@testing-library/user-event";
```
Comment (gmrabian) [inline services/ui-src/src/components/app/SkipNav.test.tsx:2]: can delete?

**Review by gmrabian — COMMENTED** [review]
It won't let me upload the video right now, but I'm seeing a weird occurrence where the first time I get to a report page, the first skip it offers is to main content instead of the top level one to report sidebar (so focus starts at the sidebar?) When you cycle around it then gets to the top level with 'skip to report sidebar' I can't remember if this is expected.
Also is it typical to have the skips located at the top always? It might be a little confusing to accept one skip and have it stay where it is with different text, but I'm also not used to tabbing around so maybe that's a new user thing

File: services/ui-src/src/components/app/SkipNav.test.tsx (hunk 1 above)
Comment (braxex) [reply services/ui-src/src/components/app/SkipNav.test.tsx:2]: yep deleted

**Comment by braxex** [comment]
@gmrabian 
>It won't let me upload the video right now, but I'm seeing a weird occurrence where the first time I get to a report page, the first skip it offers is to main content instead of the top level one to report sidebar (so focus starts at the sidebar?) When you cycle around it then gets to the top level with 'skip to report sidebar' I can't remember if this is expected.

Watched the video you sent but I'm unable to replicate locally. 

>Also is it typical to have the skips located at the top always? It might be a little confusing to accept one skip and have it stay where it is with different text, but I'm also not used to tabbing around so maybe that's a new user thing

I'm not really sure. My first thought was to have it absolute positioned with the sidebar, but when you scroll the sidebar it disappears off the page and that's not very helpful. So I went with fixed position at the top 'cause at least then it's always visible. 🤷

**Review by gmrabian — APPROVED** [review]

**Review by ntsummers1 — APPROVED** [review]

**Pull request merged**